diff -ruN libosinfo-1.12.0/osinfo/osinfo_loader.c libosinfo-1.12.0-patched/osinfo/osinfo_loader.c
--- libosinfo-1.12.0/osinfo/osinfo_loader.c	2024-10-29 14:28:59.000000000 -0300
+++ libosinfo-1.12.0-patched/osinfo/osinfo_loader.c	2026-07-26 13:39:44.355395681 -0300
@@ -354,7 +354,7 @@
         xmlXPathFreeObject(obj);
         OSINFO_LOADER_SET_ERROR(err, "Cannot format stylesheet");
     }
-    ret = g_strdup((char *)buf->content);
+    ret = g_strdup((char *)xmlBufferContent(buf));
 
     xmlBufferFree(buf);
     xmlXPathFreeObject(obj);
@@ -1902,28 +1902,32 @@
 }
 
 static void
-catchXMLError(void *ctx, const char *msg ATTRIBUTE_UNUSED, ...)
+catchXMLError(void *ctx, const char *msg, ...)
 {
     xmlParserCtxtPtr ctxt = (xmlParserCtxtPtr) ctx;
+    const xmlError *xmlErr = NULL;
+    g_autofree gchar *xmlmsg = NULL;
 
-    if (ctxt && ctxt->_private) {
-        GError **err = ctxt->_private;
-        if (!error_is_set(err)) {
-            gchar *xmlmsg;
-            if (ctxt->lastError.file) {
-                xmlmsg = g_strdup_printf("%s:%d: %s",
-                                         ctxt->lastError.file,
-                                         ctxt->lastError.line,
-                                         ctxt->lastError.message);
-            } else {
-                xmlmsg = g_strdup_printf("at line %d: %s",
-                                         ctxt->lastError.line,
-                                         ctxt->lastError.message);
-            }
-            OSINFO_LOADER_SET_ERROR(ctxt->_private, xmlmsg);
-            g_free(xmlmsg);
-        }
+    if (!ctxt || !ctxt->_private)
+        return;
+
+    if (error_is_set(ctxt->_private))
+        return;
+
+    if (!(xmlErr = xmlCtxtGetLastError(ctx)))
+        return;
+
+    if (xmlErr->file) {
+        xmlmsg = g_strdup_printf("%s:%d: %s",
+                                 xmlErr->file,
+                                 xmlErr->line,
+                                 xmlErr->message);
+    } else {
+        xmlmsg = g_strdup_printf("at line %d: %s",
+                                 xmlErr->line,
+                                 xmlErr->message);
     }
+    OSINFO_LOADER_SET_ERROR(ctxt->_private, xmlmsg);
 }
 
 static void osinfo_loader_process_xml(OsinfoLoader *loader,
