diff -ruN karchive-6.15.0/src/kar.cpp karchive-6.15.0-patched/src/kar.cpp
--- karchive-6.15.0/src/kar.cpp	2025-06-13 06:34:32.000000000 -0300
+++ karchive-6.15.0-patched/src/kar.cpp	2026-02-14 22:39:22.931897449 -0300
@@ -87,7 +87,7 @@
         return true;
     }
     if (mode != QIODevice::ReadOnly && mode != QIODevice::ReadWrite) {
-        setErrorString(tr("Unsupported mode %1").arg(mode));
+        setErrorString(tr("Unsupported mode %1").arg(int(mode)));
         return false;
     }
 
diff -ruN karchive-6.15.0/src/karchive.cpp karchive-6.15.0-patched/src/karchive.cpp
--- karchive-6.15.0/src/karchive.cpp	2025-06-13 06:34:32.000000000 -0300
+++ karchive-6.15.0-patched/src/karchive.cpp	2026-02-14 22:38:08.354898587 -0300
@@ -159,7 +159,7 @@
     }
 
     if (!d->dev->isOpen() && !d->dev->open(mode)) {
-        setErrorString(tr("Could not open device in mode %1").arg(mode));
+        setErrorString(tr("Could not open device in mode %1").arg(int(mode)));
         return false;
     }
 
@@ -205,7 +205,7 @@
         }
         break; // continued below
     default:
-        setErrorString(tr("Unsupported mode %1").arg(d->mode));
+        setErrorString(tr("Unsupported mode %1").arg(int(d->mode)));
         return false;
     }
     return true;
diff -ruN karchive-6.15.0/src/krcc.cpp karchive-6.15.0-patched/src/krcc.cpp
--- karchive-6.15.0/src/krcc.cpp	2025-06-13 06:34:32.000000000 -0300
+++ karchive-6.15.0-patched/src/krcc.cpp	2026-02-14 22:38:49.871897953 -0300
@@ -114,7 +114,7 @@
         return true;
     }
     if (mode != QIODevice::ReadOnly && mode != QIODevice::ReadWrite) {
-        setErrorString(tr("Unsupported mode %1").arg(mode));
+        setErrorString(tr("Unsupported mode %1").arg(int(mode)));
         return false;
     }
 
