diff -ruN btop-1.4.7/Makefile btop-1.4.7-patched/Makefile
--- btop-1.4.7/Makefile	2026-05-01 13:05:43.000000000 -0300
+++ btop-1.4.7-patched/Makefile	2026-07-25 01:36:37.246899196 -0300
@@ -304,9 +304,6 @@
 	@mkdir -p $(DESTDIR)$(PREFIX)/bin
 	@cp -p $(TARGETDIR)/btop $(DESTDIR)$(PREFIX)/bin/btop
 	@chmod 755 $(DESTDIR)$(PREFIX)/bin/btop
-	@printf "\033[1;92mInstalling doc to: \033[1;97m$(DESTDIR)$(PREFIX)/share/doc/btop\033[0m\n"
-	@mkdir -p $(DESTDIR)$(PREFIX)/share/doc/btop
-	@cp -p README.md $(DESTDIR)$(PREFIX)/share/doc/btop
 	@mkdir -p $(DESTDIR)$(PREFIX)/share/btop
 	@printf "\033[1;92mInstalling themes to: \033[1;97m$(DESTDIR)$(PREFIX)/share/btop/themes\033[0m\n"
 	@cp -pr themes $(DESTDIR)$(PREFIX)/share/btop
